src/BuildingBlocks/Ddd/Domain/Masa.BuildingBlocks.Ddd.Domain/_Imports.cs

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) MASA Stack All rights reserved.
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
-global using Masa.BuildingBlocks.Data.Contracts.DataFiltering;
-global using Masa.BuildingBlocks.Data.Contracts.Paginated;
+global using Masa.BuildingBlocks.Data.Contracts;
+global using Masa.BuildingBlocks.Data.Contracts;
```

Comment:
The using directive for 'Masa.BuildingBlocks.Data.Contracts' appeared previously in this namespace